src/spec-node/collectionCommonUtils/outdated.ts

```diff
@@ -0,0 +1,22 @@
+import { Argv } from 'yargs';
+import { UnpackArgv } from '../devContainersSpecCLI';
+import { outdated } from './outdatedCommandImpl';
+
+export function outdatedOptions(y: Argv) {
+	return y.options({
+		'user-data-folder': { type: 'string', description: 'Host path to a directory that is intended to be persisted and share state between sessions.' },
```

Comment:
Thinking ahead to integrating with Dependabot, I'm thinking we may potentially want to structure this change with another parser file to complement `feature_dependency_parser.rb`? (https://github.com/dependabot/dependabot-core/blob/80fdb5a0c5eafe34f8f8e87e516ae834c585508e/devcontainers/lib/dependabot/devcontainers/file_parser/feature_dependency_parser.rb#L63)

So maybe adding a flag in to only compute the _either_ outdated Features or Images might be useful?

Reply:
Good point, updated!